### PR TITLE
Add protoc-gen-swagger

### PR DESCRIPTION
**Pull request by asford**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by asford** [comment]
@conda-forge/help-python This is ready for review. It is packaged from a source archive, as there is no sdist available.

File: recipes/protoc-gen-swagger/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "protoc-gen-swagger" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Only uploaded to pypi as a .whl, not .sdist, so package is specified as github source archive
+  url: https://github.com/universe-proton/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```
Comment (BastianZim) [inline recipes/protoc-gen-swagger/meta.yaml:10]:
```suggestion
  url: https://github.com/universe-proton/protoc-gen-swagger/archive/refs/tags/v{{ version }}.tar.gz
```

Extra variables here can sometimes trip up the bot.

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just one small thing and this is ready to go.

File: recipes/protoc-gen-swagger/meta.yaml (hunk 1 above)
Comment (asford) [reply recipes/protoc-gen-swagger/meta.yaml]: Thanks!

**Review by BastianZim — APPROVED** [review]

**Pull request merged**